xml/favicons.xml

```diff
@@ -2328,4 +2336,4 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:helpsystems:goanywhere_managed_file_transfer:-"/>
   </fingerprint>
 
-</fingerprints>
\ No newline at end of file
+</fingerprints>
```

Comment:
It appears your editor added a newline at the end of the file. Please remove for consistency across fingerprint files.